src/complete.test.js

```diff
@@ -0,0 +1,20 @@
+/** ****************************************
+ * Project: js-todo-list
+ * File: complete.test.js
+ * Created: 3/9/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ ****************************************** */
+import Apps from './modules/Apps';
+import Todo from './modules/Todo';
+import Status from './modules/Status';
+
+const apps = new Apps();
+
+describe('Mark Completed', () => {
+  apps.saveItems(new Todo('Jest Test', false, 1));
+  const $items = apps.getItems();
+  Status($items, 1, true);
+  it('Item mark as complete', () => {
+    expect($items[0].completed).toBeTruthy();
+  });
+});
```

Comment:
Nice work creating the tests for the clear completed function. However, as part of the project requirements, you are to use mock for all functions that used localStorage, DOM or external APIs. Since the clear completed function uses the localStorage (the saveItems function on line 14 affects the local storage), kindly create a mock for the testing. :pray: